internal/driver/commands.go

```diff
@@ -196,10 +196,16 @@ var pprofVariables = variables{
 		"Matching includes the function name, filename or object name.")},
 	"tagfocus": &variable{stringKind, "", "", helpText(
 		"Restrict to samples with tags in range or matched by regexp",
-		"Discard samples that do not include a node with a tag matching this regexp.")},
+		"Discard samples that do not include a node with a tag matching the regexp (or within range).",
```

Comment:
I can see for all other options the description text tries to stay short for each line (~80 chars). Can we do the same. Try to make it shorter please. I'd try to just bring more examples, without even explaining them in details, I think most of them are obvious.